dangerzone/gui/__init__.py

```diff
@@ -49,9 +49,15 @@ def monkeypatch_event(arg__1: QtCore.QEvent) -> bool:
 
 
 @click.command()
-@click.argument("filename", required=False, callback=args.validate_input_filename)
+@click.argument(
+    "filenames",
+    required=False,
+    nargs=-1,
+    type=click.UNPROCESSED,
+    callback=args.validate_input_filenames,
+)
```

Comment:
At some point I tried doing that by moving the click.arguments to `dangerzone/__init__.py` but then running the tests got quite complicated -- the `DANGERZONE_MODE` env var is not set when we run the `CliRunner` and so it would just use the default one.

Or something along these lines. Can't remember exactly.

Reply:
Got it. It's not worth the extra hassle, since it's trivial.